src/index.js

```diff
@@ -133,7 +135,7 @@ module.exports = {
               message: 'Which kind of Dynamo Stream event do you want to invoke?',
               choices: [ 'INSERT', 'MODIFY', 'REMOVE' ]
             })
-            payload = mock.tablesStreams(eventName)
+            payload = mock.tablesStreams(eventName, marshallJson(mocks[pragma][name][eventName]))
```

Comment:
I believe at this moment we cannot guarantee that `mocks[pragma][name][mockName]` exists. So this probably needs to be optionally chained and generally better designed to accommodate cases where data is not present.